tekton/release-cheat-sheet.md

```diff
@@ -90,21 +87,21 @@ the pipelines repo, a terminal window and a text editor.
             name: create-draft-release
           params:
           - name: release-name
-            value: # UPDATE THIS. Example: "Ragdoll Norby"
+            value: # Example: "Ragdoll Norby"
```

Comment:
instead of creating the taskrun directly, i think following the same pattern as the other run and using env vars + tkn is a bit easier, i.e. as described at: https://github.com/tektoncd/plumbing/tree/master/tekton/resources/release#create-draft-release